flink/json-jobmanager.yml

```diff
@@ -0,0 +1,224 @@
+type: json
+data:
+  url: /v1/jobmanager/metrics?get=numRunningJobs,numRegisteredTaskManagers,taskSlotsAvailable,taskSlotsTotal,Status.JVM.Memory.Heap.Max,Status.JVM.Memory.Heap.Used,Status.JVM.Memory.Heap.Committed,Status.JVM.Memory.NonHeap.Max,Status.JVM.Memory.NonHeap.Used,Status.JVM.Memory.NonHeap.Committed,Status.JVM.Memory.Direct.Count,Status.JVM.Memory.Direct.MemoryUsed,Status.JVM.Memory.Direct.TotalCapacity,Status.JVM.Memory.Mapped.Count,Status.JVM.Memory.Mapped.MemoryUsed,Status.JVM.Memory.Mapped.TotalCapacity,Status.JVM.Threads.Count,Status.JVM.CPU.Time,Status.JVM.CPU.Load,Status.JVM.ClassLoader.ClassesLoaded,Status.JVM.ClassLoader.ClassesUnloaded
+  server: ${FLINK_HOSTPORT}
+
+observation:
+  - name: jobmanager_registered_taskmanagers_num
+    metricNamespace: flink
+    path: $.[?(@.id=numRegisteredTaskManagers)]
+    metric:
+      - name: jobmanager.registered.taskmanagers.num
+        source: value
+        type: long_gauge
+        label: num task managers
+        description: The number of registered taskmanagers
+
+  - name: jobmanager_jobs_running
+    metricNamespace: flink
+    path: $.[?(@.id=numRunningJobs)]
+    metric:
+      - name: jobmanager.jobs.running
+        source: value
+        type: long_gauge
+        label: running jobs
+        description: The number of running jobs
+
+  - name: jobmanager_memory_task_slots_total
+    metricNamespace: flink
+    path: $.[?(@.id=taskSlotsTotal)]
+    metric:
+      - name: jobmanager.memory.task.slots.total
+        source: value
+        type: long_gauge
+        label: total task slots
+        description: The total number of task slots
+
+  - name: jobmanager_memory_task_slots_available
+    metricNamespace: flink
+    path: $.[?(@.id=taskSlotsAvailable)]
+    metric:
+      - name: jobmanager.memory.task.slots.available
+        source: value
+        type: long_gauge
+        label: available task slots
+        description: The number of available task slots
+
+  - name: jobmanager_heap_max
+    metricNamespace: flink
+    path: $.[?(@.id=Status.JVM.Memory.Heap.Max)]
+    metric:
+      - name: jobmanager.heap.max
```

Comment:
Rename to `jobmanager.memory.heap.max`, also add `memory` part to all other memory related metrics.

Reply:
DONE. for both heap & nonheap